etc/tinyproxy.conf.in

```diff
@@ -234,6 +234,8 @@ Allow 127.0.0.1
 # traffic, as Tinyproxy has no control over what headers are exchanged.
 #
 #AddHeader "X-My-Header" "Powered by Tinyproxy"
+AddHeader "Connection" "Upgrade"
+AddHeader "Upgrade" "WebSocket"
```

Comment:
Is this right for requests that should not be using web sockets? I.e. external web pages.

Reply:
We don't need this change. Tiny proxy automatically upgrades to WebSocket in case of ws:// request.